packages/api/app/Controllers/Http/RoleController.js

```diff
@@ -85,6 +85,28 @@ class RoleController {
 
 		return response.status(200).send({ success: true });
 	}
+
+	/**
+	 * Delete many roles with array of id.
+	 * DELETE roles?ids=0,0,0
+	 */
+	async destroyMany({ request, response }) {
+		const { ids } = request.params;
+		const result = await Role.query()
+			.whereIn('id', ids)
+			.delete();
+		if (!result) {
+			return response
+				.status(400)
+				.send(
+					errorPayload(
+						errors.RESOURCE_DELETED_ERROR,
+						request.antl('error.resource.resourceDeletedError'),
+					),
+				);
+		}
+		return response.status(200).send({ success: true });
+	}
```

Comment:
Se puder testar o caso de erro também para cobrirmos essa linha, imagino que seria de bom proveito. Precisará envolver o método num try catch e então dar um assertError.

```suggestion
	async destroyMany({ request, response }) {
		try {
	    	const { ids } = request.params;
			const result = await Role.query().whereIn('id', ids).delete();

			if (!result) {
				throw new Error(
					errorPayload(
					errors.RESOURCE_DELETED_ERROR,
					request.antl('error.resource.resourceDeletedError')
					)
				);
			}

			return response.status(200).send({ success: true });
		} catch (error) {
			return response.status(400).json(error.message);
		}
	}
```

Reply:
Fala da linha 94?

Reply:
Olha a nova forma de lançar o erro nesse exemplo que sugeri

Reply:
Agora entendi !!

Reply:
Não precisa resolver isso agora. O benefício seria aumentar o coverage podendo testar também a failure. Se só isso impede de aprovar a PR, pode partir pras próximas issues :slightly_smiling_face::+1: